src/components/DynamicDimension/ItemSelector.js

```diff
@@ -95,13 +97,16 @@ const ItemSelector = ({
     const [state, setState] = useState({
         filter: '',
         selected: initialSelected,
+        // FIXME: keeping selected in state is redundant, use the initialSelected prop directly instead
```

Comment:
Do these FIXME need to be addressed in this PR?

Reply:
This for the `DynamicDimension`, I found this issue as the `DataDimension` was built upon the `DynamicDimension` and they thus share the same issue. I added the `FIXME` reminder for future reference, but I thought the actual fix was out of scope for this PR.

Reply:
👍